### PR TITLE
Enable option to build x64 bit VSIX

### PR DESCRIPTION
**Pull request by yongyan-gh**
Make the Sarif VS extensions to be available for 64 bit VS2022. 

image

Fix for #391

### CODE REVIEW

File: src/NuGet.Config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
```
Comment (eddynaka) [inline src/NuGet.Config:3]: before <add>, add <clear />

File: src/Sarif.Sarifer/Sarif.Sarifer.csproj [hunk 2]
```diff
@@ -51,6 +51,11 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.VSSDK.BuildTools">
+      <Version>17.0.38-dev17-g08fcf558</Version>
```
Comment (eddynaka) [inline src/Sarif.Sarifer/Sarif.Sarifer.csproj:55]: >17.0.38-dev17-g08fcf558 [](http://example.com/codeflow?start=15&length=23)

the new version is available: [https://www.nuget.org/packages/Microsoft.VSSDK.BuildTools/](https://www.nuget.org/packages/Microsoft.VSSDK.BuildTools/)

Comment (eddynaka) [reply src/Sarif.Sarifer/Sarif.Sarifer.csproj:55]: you can  also remove the nuget.config

Comment (yongyan-gh) [reply src/Sarif.Sarifer/Sarif.Sarifer.csproj]: removed

Comment (yongyan-gh) [reply src/Sarif.Sarifer/Sarif.Sarifer.csproj]: updated to official version

---
In reply to: [637965164](https://github.com/microsoft/sarif-visualstudio-extension/pull/397#discussion_r637965164) [](ancestors = 637965164)

File: src/NuGet.Config (hunk 1 above)
Comment (yongyan-gh) [reply src/NuGet.Config]: this file is no longer required and removed.

---
In reply to: [633923991](https://github.com/microsoft/sarif-visualstudio-extension/pull/397#discussion_r633923991) [](ancestors = 633923991)

**Review by eddynaka — APPROVED** [review]

**Pull request merged**